### PR TITLE
Redux Persist

### PR DESCRIPTION
**Pull request by wladyslawmajdanik**
- LastActiveDate from redux state
- Redux Persist for userReducer

### CODE REVIEW

File: src/MainTabScreen.tsx [hunk 1]
```diff
@@ -10,7 +11,12 @@ const Tab = createBottomTabNavigator<Types.MainTabParams>();
 
 export const MainTabScreen = () => {
   return (
-    <Tab.Navigator initialRouteName={Types.Route.UserProfile}>
+    <Tab.Navigator
+      initialRouteName={Types.Route.UserProfile}
+      screenOptions={{
+        tabBarHideOnKeyboard: Platform.OS !== 'ios',
```
Comment (bklukaczewski) [inline src/MainTabScreen.tsx:17]: not sure if it's needed

**Review by bklukaczewski — APPROVED** [review]

File: src/MainTabScreen.tsx (hunk 1 above)
Comment (wladyslawmajdanik) [reply src/MainTabScreen.tsx:17]: > 
Zrzut ekranu 2022-02-17 o 14 19 04


Do you have any suggestions how to fix it without hiding tabBar?
 I tried changing windowSoftInputMode but it didn't helped.

Comment (bklukaczewski) [reply src/MainTabScreen.tsx:17]: maybe let's keep it true for both platforms then? :)

Comment (wladyslawmajdanik) [reply src/MainTabScreen.tsx:17]: with true on IOS i have problem with IOS keyboard ;(

**Pull request merged**